android/java/org/chromium/chrome/browser/appmenu/BraveShieldsMenuHandler.java

```diff
@@ -265,7 +265,7 @@ private void runMenuItemEnterAnimations() {
         ViewGroup list = mPopup.getListView();
         for (int i = 0; i < list.getChildCount(); i++) {
             View view = list.getChildAt(i);
-            Object animatorObject = view.getTag(R.id.menu_item_enter_anim_id);
+            Object animatorObject = null;/* = view.getTag(R.id.menu_item_enter_anim_id);*/
```

Comment:
?

Reply:
These are commented out just to make it build. There's a follow up issue for the Android team to fix:
brave/brave-browser#7726